tests/cp2/test_cp2_cbez.s

```diff
@@ -49,6 +51,7 @@ L1:
 		nop
 		dli	$a1, 0
 		cgetdefault	$c22
+		cincoffset	$c22, $c22, 1	# Ensure non-zero address
```

Comment:
Ehh, technically you could go from -1 to 0.... :P but we really should have the offset as 0 by this point so it's fine